### PR TITLE
feat: support recent blocks, block details in sequencer tier

### PR DESCRIPTION
**Pull request by evilpeach**
Fixes: CFE-531, CFE-535

### CODE REVIEW

File: src/lib/services/tx/sequencer.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import type { Option } from "lib/types";
+import { parseWithError } from "lib/utils";
+
+export const getTxsByBlockHeightSequencer = async (
+  endpoint: string,
+  height: number,
+  paginationKey: Option<string>
+) =>
+  axios
+    .get(`${endpoint}/indexer/tx/v1/txs/by_height/${height}`, {
```
Comment (Poafs1) [inline src/lib/services/tx/sequencer.ts:13]: `encodeURI`

**Review by Poafs1 — DISMISSED** [review]
look good

File: src/lib/pages/blocks/index.tsx [hunk 2]
```diff
@@ -26,7 +27,11 @@ const BlocksPage = () => {
         subtitle="This page displays all blocks on this network sorted by recency"
         docHref="introduction/overview#recent-blocks"
       />
-      <RecentBlocksTable />
+      <TierSwitcher
+        full={<RecentBlocksTableFull />}
+        sequencer={<RecentBlocksTableSequencer />}
+        lite={null}
```
Comment (songwongtp) [inline src/lib/pages/blocks/index.tsx:33]: make it optional like `sequencer`?

File: src/lib/services/types/block.ts [hunk 3]
```diff
@@ -21,18 +21,23 @@ import { zTx } from "./tx";
 const zNullableValidator = z.nullable(
   z
     .object({
-      validator_address: zValidatorAddr.nullable(),
+      validator_address: zValidatorAddr.nullish(),
+      operator_address: zValidatorAddr.nullish(),
```
Comment (songwongtp) [inline src/lib/services/types/block.ts:25]: isn't this the same as validator_address? do we really need to have 2 fields for the same data?

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/services/types/block.ts [hunk 4]
```diff
@@ -146,3 +148,41 @@ export const zBlockDataResponseLcd = zBlockLcd
...
+  gas_used: z.string(),
+  gas_wanted: z.string(),
+  tx_count: z.string(),
+  proposer: zNullableValidator,
+});
+
+const zBlocksResponseItemSequencer = zBlockSequencer.transform<Block>(
+  (val) => ({
+    hash: val.hash,
+    height: Number(val.height),
```
Comment (songwongtp) [inline src/lib/services/types/block.ts:166]: Number => coerce

**Review by songwongtp — APPROVED** [review]

**Review by Poafs1 — COMMENTED** [review]
lgtm

**Pull request merged**